### PR TITLE
Restore test with factory-girl

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts [hunk 1]
```diff
@@ -1,122 +1,90 @@
...
+    alice = await factory.create<ReturnType<typeof identityActions.addNewIdentity>['payload']>(
+      'Identity',
+      { id: community.id, zbayNickname: 'alice' }
+    )
+  })
+
+  test('ask for missing messages', async () => {
+    const types = [MessageType.Basic, MessageType.Empty, MessageType.Empty]
+
+    const shuffled = types
```
Comment (holmesworcester) [inline packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts:48]: Curious: why are we shuffling?

File: packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts [hunk 2]
```diff
@@ -1,122 +1,90 @@
...
-            )
-          }
-        }
-      )
+          verifyAutomatically: true
+        })
+      })
+    )
+
+    const empties = messages.filter(item => item.message.type === MessageType.Empty)
```
Comment (holmesworcester) [inline packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts:73]: Where are empty messages from?

Comment (holmesworcester) [reply packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts:73]: We're just not displaying empty messages?

**Review by holmesworcester — APPROVED** [review]

File: packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts (hunk 1 above)
Comment (siepra) [reply packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts:48]: It probably not necessary but I wanted to make sure the mechanism will never depend on the messages order

File: packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts (hunk 2 above)
Comment (siepra) [reply packages/nectar/src/sagas/publicChannels/checkForMessages/checkForMessages.saga.test.ts:73]: We add message object with no data but the id, as we receive the list of ids from waggle. Then we ask for the messages and we update those 'empty' objects with data

**Review by EmiM — APPROVED** [review]

**Pull request merged**